pkl-gradle/src/main/java/org/pkl/gradle/task/EvalTask.java

```diff
@@ -67,17 +67,18 @@ public abstract class EvalTask extends ModulesTask {
                               getMultipleFileOutputDir(), it -> it.getAsFile().getAbsolutePath()),
                           getExpression().get())));
 
+  @SuppressWarnings("unused")
   @OutputFiles
   @Optional
-  public FileCollection getEffectiveOutputFilePaths() {
+  public FileCollection getEffectiveOutputFiles() {
     return getObjects()
         .fileCollection()
         .from(cliEvaluator.map(e -> nullToEmpty(e.getOutputFiles())));
   }
 
   @OutputDirectories
   @Optional
-  public FileCollection getOutputDirectories() {
+  public FileCollection getEffectiveOutputDirectories() {
```

Comment:
Minor - I believe in all of the plugins we use `getSomeDir()` for directory properties, not `getSomeDirectory()`, so it might make sense to name this `getEffectiveOutputDirs()`. WDYT?

Reply:
Non-blocking, but I agree with @netvl.